integrations/newrelic/.port/resources/port-app-config.yaml

```diff
@@ -90,3 +90,32 @@ resources:
             tags: .tags
           relations:
             newRelicService: .serviceLevel.indicators[0].guid
+  - kind: entity
+    selector:
+      query: 'true'
+      infrastructureIntegrationTypes:
+        - AWS_EC2_INSTANCE
+        - AWS_S3_BUCKET
+        - AWS_RDS_DB_INSTANCE
+        - AWS_LAMBDA_FUNCTION
+        - AWS_ELB_LOAD_BALANCER
+        - AZURE_VIRTUAL_MACHINE
+        - AZURE_SQL_DATABASE
+        - GCP_COMPUTE_INSTANCE
+        - GCP_STORAGE_BUCKET
+        - GCP_SQL_DATABASE_INSTANCE
+        # Add more infrastructure integration types as needed
```

Comment:
based on this - https://github.com/port-labs/ocean/blob/7e7688363271ac49cc2f6c6ee8e48849494c714b/integrations/newrelic/newrelic_integration/overrides.py#L17
I don't see we have this kind of option in the mapping, was this tested?

Reply:
are you referring to `newrelic_types`?